src/core/dbus-cgroup.c

```diff
@@ -2053,6 +2054,38 @@ int bus_cgroup_set_property(
                 return 1;
         }
 
+        if (streq(name, "ManagedOOMMemoryPressureDurationUSec")) {
+                uint64_t t;
+
+                if (!UNIT_VTABLE(u)->can_set_managed_oom)
+                        return sd_bus_error_setf(error, SD_BUS_ERROR_INVALID_ARGS, "Cannot set %s for this unit type", name);
+
+                r = sd_bus_message_read(message, "t", &t);
+                if (r < 0)
+                        return r;
+
+                /* Zero is treated as default value (USEC_INFINITY) in D-Bus API. */
+                t = t ?: USEC_INFINITY;
```

Comment:
Is this necessary? I'd like to drop this.

Reply:
This is necessary to make 0 work in dbus API per your initial proposal:

> dbus method accepts zero and 1sec <= t <= infinity.

If we don't want 0 to work (which I think is probably fine), then we can drop this

Reply:
Now, getting dbus property always provides >= 1sec (including infinity). So, I think setting dbus property should not support any values < 1sec especially 0.